functions/go/fix/fixpkg/fix.go

```diff
@@ -330,10 +390,15 @@ func (s *Fix) FixKptfile(node *yaml.RNode, functions []v1alpha2.Function) (*yaml
 	}
 
 	if len(setters) > 0 {
-		fn := v1alpha2.Function{
-			Image:     "gcr.io/kpt-fn/apply-setters:v0.1",
-			ConfigMap: setters,
+		fn := v1.Function{
+			Image:      "gcr.io/kpt-fn/apply-setters:v0.1",
+			ConfigPath: "setters-config.yaml",
```

Comment:
@phanimarupaka Instead of calling this file `setters-config.yaml` we should just call it `setters.yaml`. This would make it analogous to `values.yaml` in Helm or `terraform.tfvars` in Terraform. In general, it's *all* config.

Reply:
+1. I flagged this in the fishfood as well. In general, "config" and "resource" are meaningless names.

Reply:
Will do.